cnf-certification-test/lifecycle/suite.go

```diff
@@ -36,19 +37,103 @@ var _ = ginkgo.Describe(common.LifecycleTestKey, func() {
 		provider.BuildTestEnvironment()
 		env = provider.GetTestEnvironment()
 	})
+	testContainersPreStop(&env)
+	testContainersImagePolicy(&env)
+	testContainersReadinessProbe(&env)
+	testContainersLivenessProbe(&env)
+	testPodsOwnerReference(&env)
+})
+
+func testContainersPreStop(env *provider.TestEnvironment) {
 	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestShudtownIdentifier)
 	ginkgo.It(testID, ginkgo.Label(testID), func() {
-		gomega.Expect(true).To(gomega.Equal(true))
 		badcontainers := []string{}
 		for _, cut := range env.Containers {
-			fmt.Println("container ", cut.Data.Name)
-			logrus.Debugln("check container ", cut.Data.Name) // maybe use different platform ?
+			logrus.Debugln("check container ", cut.Namespace, " ", cut.Podname, " ", cut.Data.Name, " pre stop lifecycle ")
 			if cut.Data.Lifecycle.PreStop == nil {
 				badcontainers = append(badcontainers, cut.Data.Name)
 				logrus.Errorln("container ", cut.Data.Name, " does not have preStop defined")
```

Comment:
I'd use ClaimFilePrintf here instead of logrus. Also, add the pod name and namespace so it has more information so the customer can find this target easier.